index.html

```diff
@@ -9,7 +9,36 @@
 {% include lightbendBanner.html %}
 {% include topbar.html %}
 
-<section class="wrapper hero">
+<section class="wrapper hero-bsl">
+    <div class="row">
+        <div id="hero-bsl-left-col" class="sixcol">
+            <h2 class="bright">Do You Have Your Akka BSL License?</h2>
+            <p>A license ensures the version of Akka (beginning with v2.7 released Sept 2022) you
```

Comment:
```suggestion
            <p>A license ensures the version of Akka (beginning with version 2.7, released in September 2022) you
```
The sentence seems a little unnecessarily complicated.

Reply:
Change commited